### ISSUE TITLE
Could not find a declaration file for module 'sveltekit-headless-table'

### ISSUE THREAD

**Issue by jmjaimesmendoza** [bryanmylee/svelte-headless-table#157]
When following the Quick Start section on the docs while using svelte (just svelte, no sveltekit) i get the following error:
```
Could not find a declaration file for module 'svelte-headless-table'. '/Users/juanmartinjaimesmendoza/tracker/node_modules/.pnpm/svelte-headless-table@0.17.5_svelte@4.2.1/node_modules/svelte-headless-table/index.js' implicitly has an 'any' type.
```
any clues as to how to solve this?

**Comment by gpaiva985** [comment]
I got the same error when trying to use it with shadcn-svelte.
I solve it by not installing it as a devDependency (e.g. pnpm install svelte-headless-table)

**Comment by pitzzahh** [comment]
Same issue here

**Comment by hgoona** [comment]
Is this why I'm getting this kind of error/warning in VSCODE, using SvelteKit? >>>
![image](https://github.com/bryanmylee/svelte-headless-table/assets/30910108/51140b11-0790-4c39-9463-882aa8f87c74)


I installed the library using `pnpm install svelte-headless-table`. I'd like to get rid of the error / set the Types for my column creating function ?correctly ...

**Comment by shyakadavis** [comment]
Hi. Wanted to confirm I'm getting the same error as well.

```
Could not find a declaration file for module 'svelte-headless-table'. '.../node_modules/.pnpm/svelte-headless-table@0.17.7_svelte@4.2.7/node_modules/svelte-headless-table/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/svelte-headless-table` if it exists or add a new declaration (.d.ts) file containing `declare module 'svelte-headless-table';`ts(7016)
```

The proposed workaround of installing it as a dependency is NOT working for me. This is very odd.

For context, I'm also using it under the `shadcn-svelte` umbrella, and `SvelteKit`.

Hope @bryanmylee could provide some input on this. 🙂

**Comment by ftognetto** [comment]
Same problem here, there must be a dependency problem, in other project I do not have this issue.
Tried to change svelte / SvelteKit / Vite version without solving.
I also have svelte-headless-table installed as a dependency v 0.17.7

**Comment by ftognetto** [comment]
The problem is propagating across all my projects 😭
<img width="790" alt="Screenshot 2023-11-28 alle 12 02 04" src="https://github.com/bryanmylee/svelte-headless-table/assets/47910896/da207b2d-aff4-4efc-87d1-1f9372d76928">

**Comment by bryanmylee** [comment]
Fixed by #176! I'll be making a v1 release in line with this.
